core/src/main/java/org/nervos/ckb/methods/type/CellInput.java

```diff
@@ -4,44 +4,23 @@
 
 import java.util.List;
 
-/**
- * Created by duanyytop on 2019-01-08.
- * Copyright © 2018 Nervos Foundation. All rights reserved.
- */
+/** Created by duanyytop on 2019-01-08. Copyright © 2018 Nervos Foundation. All rights reserved. */
 public class CellInput {
 
-    /**
-     * previous_output : {"hash":"0x0000000000000000000000000000000000000000000000000000000000000000","index":4294967295}
-     * unlock : {"args":[],"binary":[1,0,0,0,0,0,0,0],"reference":null,"signed_args":[],"version":0}
-     */
+  @JsonProperty("tx_hash")
+  public String txHash;
```

Comment:
https://github.com/nervosnetwork/ckb/pull/533/files#diff-a048830b754aed80272a8bba2a30d2ffR67
That doesn't seem to have changed about OutPoint

Reply:
![image](https://user-images.githubusercontent.com/5823268/56640962-ca879c00-66a6-11e9-9cd2-c05261b19952.png)